### ISSUE TITLE
report sucessful rawhide builds only once

### ISSUE THREAD

**Issue by tyll** [fedora-infra/the-new-hotness#17]
If a bug about a new release is open and the package is built twice in Rawhide, the new hotness creates a comment for each build, even if the first build was successful.

**Comment by ralphbean** [comment]
Respectfully, I'd like to close this one as wontfix.

It seems to me like one of these two must be true:
- The second rawhide build is still relevant to the ticket.
- Or the owner should close the bug.

If there's disagreement, please re-open.

**Comment by tyll** [comment]
I noticed this when I rebuilt someone else's packages, so it will probably also happen a lot when there is a mass rebuild. I do not see any value in reporting the build again and also closing the ticket does not make sense. Overall it just adds more noise afaics, therefore I am still for changing this.

**Comment by bochecha** [comment]
> and also closing the ticket does not make sense.

Why not?

The ticket notifies you a new version is available, and the new version has been built into Rawhide.

Surely, the ticket can be closed with the RAWHIDE resolution?

**Comment by ralphbean** [comment]
> Surely, the ticket can be closed with the RAWHIDE resolution?

A number of people have spoken up saying that they like leaving the ticket open until Bodhi closes it later (when updates for released branches get finally pushed to stable).
